app/src/main/java/org/cru/godtools/ui/dashboard/tools/GridConcatAdapter.kt

```diff
@@ -0,0 +1,41 @@
+package org.cru.godtools.ui.dashboard.tools
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.annotation.LayoutRes
+import androidx.databinding.DataBindingUtil
+import androidx.databinding.ViewDataBinding
+import androidx.lifecycle.LifecycleOwner
+import androidx.lifecycle.Observer
+import androidx.recyclerview.widget.RecyclerView
+import org.ccci.gto.android.common.recyclerview.adapter.SimpleDataBindingAdapter
+import org.cru.godtools.R
+
+class GridConcatAdapter(
```

Comment:
`Grid` doesn't make sense in this class name, adapter's have no concern of what LayoutManager is used to display their items.

Reply:
`Concat` also does not make sense in this adapter name, because there is no logic in the adapter about concating multiple things together.